cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalTest.scala

```diff
@@ -234,24 +235,24 @@ class TemporalTest extends DocumentingTest {
           p(
             """The least significant components can be omitted.
               |For example, a time may be specified with `Hour` and `Minute`, leaving out `Second` and `fraction`.
-              |On the other hand, specifying a time with `Hour` and `Second` -- leaving out `Minute` -- is not possible.
+              |On the other hand, specifying a time with `Hour` and `Second`, while leaving out `Minute`, is not possible.
             """.stripMargin)
         }
         section("Specifying time zones", "cypher-temporal-specify-time-zone") {
           p(
-            """The <<cypher-temporal-timezones, time zone>> is specified in one of the following ways:
+            """The time zone is specified in one of the following ways:
               |
               |* As an offset from UTC
-              |* Using the "**z**" shorthand for the UTC (`±00:00`) time zone
+              |* Using the **`Z`** shorthand for the UTC (`±00:00`) time zone
               |
           """.stripMargin)
           p(
             """
             |When specifying a time zone as an offset from UTC, the rules below apply:
             |
             |* The time zone always starts with either a plus (`+`) or minus (`-`) sign.
-            | ** Positive offsets -- time zones beginning with `+` -- denote time zones east of UTC.
-            | ** Negative offsets -- time zones beginning with `-` -- denote time zones west of UTC.
+            | ** Positive offsets, or time zones beginning with `+`, denote time zones east of UTC.
```

Comment:
This isn't an "either ... or" situation, which the change to my text now indicates. A positive offset *is* a time zone beginning with `+`. My original text qualified this clearly. If the dashed method is not acceptable, I suggest an "i.e." or something along those lines.

The same goes for the negative offset text below